tls/s2n_server_extensions.c

```diff
@@ -42,13 +42,23 @@ int s2n_server_extensions_send(struct s2n_connection *conn, struct s2n_stuffer *
     if (s2n_server_can_send_ocsp(conn)) {
         total_size += 4;
     }
+    if (conn->secure_renegotiation) {
+        total_size += 5;
+    }
 
     if (total_size == 0) {
         return 0;
     }
 
     GUARD(s2n_stuffer_write_uint16(out, total_size));
 
+    /* Write the renegotiation_info extension */
+    if (conn->secure_renegotiation) {
+        GUARD(s2n_stuffer_write_uint16(out, TLS_EXTENSION_RENEGOTIATION_INFO));
+        GUARD(s2n_stuffer_write_uint16(out, 1));
+        GUARD(s2n_stuffer_write_uint8(out, 0));
```

Comment:
Can you either add a comment or create variables with descriptive names that explain what 0 and 1 are representing?